dev/reporter-options.json

```diff
@@ -0,0 +1,6 @@
+{
+    "reporterEnabled": "spec, mocha-junit-reporter",
+    "mochaJunitReporterReporterOptions": {
+        "mochaFile": "test-results.xml"
```

Comment:
I remember having problems on certain OS's if I didn't specify the full path. Does this report results on all three OS's?